src/eslint.js

```diff
@@ -39,6 +39,8 @@ async function execute(args = []) {
             if (message.severity >= 2) {
 
                 let filePath = path.relative(process.cwd(), file.filePath);
+                const sanitizedfilePath = filePath.replace('\\\\', '/');
```

Comment:
For safety I'd probably replace with a regex /(\\\\)+/ into /, so that it supports two or 4 (or 6..) backslashes